minfraud/models.py

```diff
@@ -708,13 +710,23 @@ class Subscores(object):
 
       :type: float | None
 
+      .. deprecated:: 1.8.0
+        Deprecated effective August 29, 2019. This subscore will default to 1
+        and will be removed in a future release. The IP tenure is reflected in
+        the overall risk score.
+
     .. attribute:: ip_tenure
 
       The risk associated with the issuer ID number on the IP address. If
       present, this is a value in the range 0.01 to 99.
 
       :type: float | None
 
+      .. deprecated:: 1.8.0
+        Deprecated effective August 29, 2019. This subscore will default to 1
+        and will be removed in a future release. The user tenure on email is
+        reflected in the `/subscores/email_address` output.
```

Comment:
This is reStructureText rather than markdown; you need two backticks rather than one. It might also be nice to refer to the attribute rather than the path in the JSON.